db/migrate/20201109135332_enable_pg_trgm_extension.rb

```diff
@@ -0,0 +1,5 @@
+class EnablePgTrgmExtension < ActiveRecord::Migration[6.0]
+  def change
+    enable_extension "pg_trgm"
```

Comment:
Are we using this extension or planning to use it later on?

Reply:
Actually, I don't think we need it at the moment. Have removed in https://github.com/DFE-Digital/register-trainee-teachers/pull/161/commits/83d9093f98e0da85074cf5c4e302fc49ba9369d0